manifest.toml

```diff
@@ -60,4 +60,5 @@ ram.runtime = "50M"
     webdav.auth_header = false
 
     [resources.apt]
-    packages = "sqlite3, php8.2-sqlite3, php8.2-gd, php8.2-intl, php8.2-cli, php8.2-gnupg, php8.2-mbstring, php8.2-zip"
+    packages = "sqlite3, php8.2-sqlite3, php8.2-imagick, php8.2-intl, php8.2-cli, php8.2-gnupg, php8.2-mbstring, php8.2-zip, mupdf-tools, ffmpeg"
```

Comment:
bt i replaced gd with imagick as the paheko docs says they prefer imagick: 

> Images : nécessite d'avoir l'extension __PHP Imagick (de préférence)__ ou GD.

https://fossil.kd2.org/paheko/wiki?name=Configuration/Thumbnails